src/x11/keymap.c

```diff
@@ -338,9 +338,9 @@ translate_action(union xkb_action *action, const xcb_xkb_action_t *wire)
 
         /* Treat high unknown actions as Private actions. */
         action->priv.type = wire->noaction.type;
-        STATIC_ASSERT(sizeof(action->priv.data) == 7 &&
-                      sizeof(wire->noaction.pad0) == 7,
-                      "The private action data must be 7 bytes long!");
+        _Static_assert(sizeof(action->priv.data) == 7 &&
```

Comment:
Can we use `static_assert` instead of the uglier `_Static_assert`? It should be defined in `<assert.h>` according to [here](https://en.cppreference.com/w/c/error/static_assert).

Reply:
Fixed. I missed that although not a keyword in C11, we do have this convenience macro.